site/content/en/docs/formats/ade20k2017.md

```diff
@@ -82,6 +83,9 @@ image. Each line in the text file contains:
 Each column is separated by a `#`. See example of dataset
 [here](https://github.com/openvinotoolkit/datumaro/tree/develop/tests/assets/ade20k2017_dataset).
 
+To add unformatted classes, you can use `dataset_meta.json`.
+Information about `dataset_meta.json` can be found [here](/docs/user_manual/supported_formats/#dataset-meta-file).
+
```

Comment:
```suggestion
To add unformatted classes, you can use [`dataset_meta.json`](/docs/user_manual/supported_formats/#dataset-meta-file).

```

Please find a better term for "unformatted classes", its meaning is not clear enough.

Reply:
Custom classes?